requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Base dependencies
 black
 flake8
-PySide6
+PySide6==6.6.3.1
```

Comment:
Why exactly this version and not something like `>=6.6.3.1<6.7`? This would not exclude us from further bug fixes in that series but still maintain feature parity.

Reply:
there is no version in between and 6.6.3 is also no longer being developed further